### PR TITLE
Task/die on redirect error

### PR DESCRIPTION
**Pull request by dmoranj**
Add a flag to the configuration to die on redirection error instead of returning a 501 code.

Implements #223

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -134,6 +134,12 @@ config.middlewares = {
 };
 
 /**
+ * If this flag is activated, whenever the pepProxy is not able to redirect a request, instead of returning a 501 error
+ * (that is the default functionality) the PEP Proxy process will exit with a -2 code.
+ */
+config.dieOnRedirectError = false;
```
Comment (fgalan) [inline config.js]: Maybe this should be documented in the corresponding .md file?

File: config.js [hunk 2]
```diff
@@ -134,6 +134,12 @@ config.middlewares = {
 };
```
Comment (fgalan) [inline config.js]: Is this a bugfix? Is worth the time to include in the CNR file?

Comment (dmoranj) [inline config.js]: Fixed in ab7541c

File: config.js (hunk 1 above)
Comment (dmoranj) [inline config.js]: FIxed in ab7541c

**Comment by fgalan** [comment]
LGTM

**Comment by fgalan** [comment]
Waiting for @mcarracedo feedbakc about bug severity in order to know to which branches the fix has to be merged.

**Comment by mcarracedo** [comment]
Please, merge into release/0.7.0

**Comment by mcarracedo** [comment]
LGTM

**Pull request merged**